src/renderer/pages/ProcessIssues/ReportModal.tsx

```diff
@@ -5,6 +5,20 @@ import { longMessages, shortMessages } from '../../messages/process-error';
 import Spinner from '../../assets/spinner.svg';
 import { ProcessIssue } from '../../../workers/types';
 
+const posibleErrorStates = ['ERROR', 'TO_MANY_REPORTS'] as const;
+type ErrorReportRequestState = typeof posibleErrorStates[number];
+type ReportRequestState = 'READY' | 'SENDING' | ErrorReportRequestState;
+
+const stateIsError = (maybe: unknown): maybe is ErrorReportRequestState =>
+  typeof maybe === 'string' &&
+  posibleErrorStates.includes(maybe as ErrorReportRequestState);
+
+const errorMessages: Record<ErrorReportRequestState, string> = {
+  ERROR:
+    'We could not send your request, make sure you are connected to the internet',
+  TO_MANY_REPORTS: 'You cannot create more request for today',
```

Comment:
`TOO_MANY_REPORTS`

Reply:
And a better message could be "Reached reporting errors day limit"

Reply:
done